### PR TITLE
awscli: Fix test for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by iMichka** [comment]
Should fix:
```
==> brew test awscli --verbose
Testing awscli
==> /home/linuxbrew/.linuxbrew/Cellar/awscli/1.11.138/bin/aws help

Could not find executable named "groff"
Error: awscli: failed
<0> expected but was
<255>.
==> FAILED
```

File: Formula/awscli.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Awscli < Formula
   url "https://github.com/aws/aws-cli/archive/1.11.138.tar.gz"
   sha256 "ecc411a0db77694ef7a24853bef770ca1c9e54d747a1283599ea7a3a3c1478d0"
   head "https://github.com/aws/aws-cli.git", :branch => "develop"
+  revision 1 unless OS.mac?
```
Comment (sjackman) [inline Formula/awscli.rb]: No need to bump the revision for a change like this where the existing bottle is expected to be fine. You can merge the PR without updating the bottle.

File: Formula/awscli.rb [hunk 2]
```diff
@@ -19,7 +20,10 @@ class Awscli < Formula
   # https://github.com/Homebrew/homebrew/pull/37240
   depends_on :python if MacOS.version <= :lion
 
-  depends_on "libyaml" unless OS.mac?
+  unless OS.mac?
+    depends_on "groff"
```
Comment (sjackman) [inline Formula/awscli.rb]: For unusual dependencies like this, add a comment like `# needed for aws help`

**Comment by rwhogg** [comment]
I agree with Shaun that a `groff` dependency is not necessary. It builds fine without it, it's not essential to functionality, and the error you get if you try to run the help command without it present is quite clear:

```
% aws help

Could not find executable named "groff"
```

Based on the clarity of that error message, and that I think most users will probably have `groff` available from the system anyways, I'd say that we don't even need a caveat.

**Pull request closed without merging**